input/resources/CodeSystem-name-context.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CodeSystem xmlns="http://hl7.org/fhir">
+  <id value="name-context"/>
+  <meta>
+    <profile value="http://hl7.org/fhir/StructureDefinition/shareablecodesystem"/>
+  </meta>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm">
+    <valueInteger value="0"/>
+  </extension>
+  <url value="http://terminology.hl7.org.au/CodeSystem/name-context"/>
+  <name value="NameContextCodeSystem"/>
+  <title value="Name Context Codes"/>
+  <status value="active"/>
```

Comment:
status should be draft

Reply:
I think the word "Codes" is redundant in a CodeSystem resource.